oak_runtime/src/io.rs

```diff
@@ -29,6 +29,9 @@ pub trait ReceiverExt<T> {
 
     /// Waits, reads and decodes a message from the [`Receiver::handle`].
     fn receive(&self, runtime: &RuntimeProxy) -> Result<T, OakError>;
+
+    /// Get the label associated with this receiver.
+    fn label(&self, runtime: &RuntimeProxy) -> Result<Label, OakError>;
```

Comment:
It seems this could just return `Result<Label, OakStatus>`, which is strictly more precise? Not a big deal anyways, I guess we don't do much with the error anyways.

Reply:
I agree that `OakError` does not add anything here, but all the other methods in `ReceiverExt` and `SenderExt` use `OakError` as the error type. I am using `OakError` to be consistent.